dlk/python/dlk/core/optimizer.py

```diff
@@ -355,6 +355,10 @@ def pass_compute_thresholds(graph: Graph) -> None:
                     bn_nega_idx = [v for v in range(len(bn_scale)) if bn_scale[v] < 0]
             threshold = (trans_th['data'] * np.float64(n)) / (np.float64(max_v) * scaling_factor)
 
+            # take care of threshold values that are larger than 13-bit signed integer
```

Comment:
This comment is wrong, 12 bit is correct.

https://github.com/blue-oil/blueoil/blob/7f704a253df7dadec6be05bda620f68009f3932f/dlk/python/dlk/core/optimizer.py#L337-L338

Reply:
Thanks for the comment, I thought a 13 bit signed integer has max value of 2^12 - 1 for the sign-bit being the highest bit.

Reply:
Thanks, Aha, you are right!!
And I just remember, I wrote same comment in previous PR. sorry ~